minecraft_servers/gamerherz/manifest.json

```
@@ -24,6 +24,11 @@
       "command": "/server Duels_8-1",
       "versions": "1.8<*"
     },
+    "Build Attack 4": {
```

Not a valid json key
```suggestion
    "build-attack-4": {
```